pegasus/router.rb

```diff
@@ -171,6 +171,7 @@ def self.set_max_age(type, default)
   get '*' do |uri|
     pass unless path = resolve_static('public', uri)
     cache :static
+    NewRelic::Agent.set_transaction_name(uri) if defined? NewRelic
```

Comment:
This will potentially spam NewRelic with new transaction names if anything has a session ID or any sort of unique ID in the path. 

I'm not sure if there is an intelligent way to strip out anything that's an ID. You can sub out all numbers that are 2 digits or more will work if all of our IDs are sequences.